modules/nf-core/beagle5/beagle/main.nf

```diff
@@ -8,7 +8,7 @@ process BEAGLE5_BEAGLE {
         'biocontainers/beagle:5.5_27Feb25.75f--hdfd78af_0' }"
 
     input:
-    tuple val(meta), path(vcf), path(vcf_index), path(refpanel), path(refpanel_index), path(genmap), path(exclsamples), path(exclmarkers)
+    tuple val(meta), path(vcf), path(vcf_index), path(refpanel), path(refpanel_index), path(genmap), path(exclsamples), path(exclmarkers), val(region)
```

Comment:
What is the reasoning for this being an input value? It doesn't seem mandatory to me. Optional values should always be passed to the module via the configuration